packages/react-components/react-color-picker-preview/library/src/components/ColorArea/ColorArea.types.ts

```diff
@@ -1,17 +1,46 @@
-import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import * as React from 'react';
+import type { ComponentProps, ComponentState, Slot, EventHandler, EventData } from '@fluentui/react-utilities';
+import type { Numberify, HSVA } from '@ctrl/tinycolor';
+
+export type ColorAreaOnColorChangeData = EventData<'change', React.ChangeEvent<HTMLInputElement>> & {
+  color: Numberify<HSVA>;
+};
 
 export type ColorAreaSlots = {
-  root: Slot<'div'>;
+  root: NonNullable<Slot<'div'>>;
+  thumb: NonNullable<Slot<'div'>>;
+  inputX: NonNullable<Slot<'input'>>;
+  inputY: NonNullable<Slot<'input'>>;
+};
+
+export type HsvColor = {
+  h: number;
+  s: number;
+  v: number;
+  a?: number;
 };
 
 /**
  * ColorArea Props
  */
-export type ColorAreaProps = ComponentProps<ColorAreaSlots> & {};
+export type ColorAreaProps = Omit<ComponentProps<Partial<ColorAreaSlots>>, 'onChange'> & {
+  /**
+   * The current color of the ColorArea.
+   */
+  color?: Numberify<HSVA>;
```

Comment:
What's the difference between the `Numberify<HSVA>` and `HsvColor` types? Could the `HsvColor` be used instead?